csharp.html.markdown

```diff
@@ -1299,16 +1315,27 @@ namespace Csharp7
 ```
 
 ## Topics Not Covered
+✨ New, 👍 Old, 🎈 LTS, 🔥 Cross-platform, 🎁 Windows-only
 
  * Attributes
- * async/await
+
+ * Asynchronous Programming
+
  * Web Development
- 	* ASP.NET MVC & WebApi (new)
- 	* ASP.NET Web Forms (old)
- 	* WebMatrix (tool)
+    * ASP.NET Core ✨
+
  * Desktop Development
- 	* Windows Presentation Foundation (WPF) (new)
- 	* Winforms (old)
+ 	* Windows Presentation Foundation 👍 🎈 🎁
+    * Universal Windows Platform ✨ 🎁
+    * Uno Platform 🔥 ✨
```

Comment:
Is it that new thing that supersedes UWP?

Reply:
I wouldn't say supersedes UWP. You can use WinUI 2 controls inside UWP or WPF/WinForms (Via XAML Islands) too